js/src/personal/haoking/NavLink.tsx

```diff
@@ -0,0 +1,24 @@
+import styles from './Haoking.module.css'
+
+/* Component for each individual link in Navbar */
+export function NavLink({
```

Comment:
Were you running into issues with the Mantine NavLink or Anchor components?
Or do you just want to make your own?

Reply:
I didn't know mantine provided navlink or anchor components so I ended up just making my own